vignettes/RPyGeo.Rmd

```diff
@@ -141,31 +142,35 @@ arcpy$Dissolve_management(in_features = "nz.shp",
   arcpy$PolygonToLine_management("nz_border.shp")
 ```
 
-In this example the `nz.shp` shapefile is dissolved on the `Island` field and then the polygons are converted to polylines.
+In this example the `nz.shp` shapefile is dissolved based on the `Island` field.
+Subsequently, the polygons are converted to polylines.
+
+<!-- Figure is missing a caption and a reference in the text. -->
 
 ![NZ Workflow.](nz.png){width=100%}
 
 ### Modules and extensions
 
-RPyGeo imports all functions and classes of the available ArcPy modules and extensions. 
-You can access all geoprocessing tools found in the standard toolboxes with a single `$` operator. 
-To access the functions and classes of another module, you have to use a second `$` operator to specify the module.
+To use the functions of a specific extension, you have to first access the corresponding extension via the `$` operator, and then have to use a second `$`  to get access to its functions.
+For example, to compute the slope from a digital elevation model with the help of the Spatial Analyst extenstion, we run:
 
 ```{r, eval=FALSE}
 arcpy$sa$Slope(in_raster = "elev.tif")
 ```
 
-The `Slope` function of the Spatial Analysis extension is located under `sa`. 
+Note that there is no output parameter, hence the result of the processing will be saved to the scratch workspace location (see again `rpygeo_build_env()`).
 
 ### Load output file into R Session
 
-The output of an ArcPy function can be directly loaded into the R session with the `rpygeo_load` function.
+The output of an ArcPy function can be directly loaded into the R session with the `rpygeo_load()` function.
 
+<!-- well, I would never write that... a right-handed assignment operator-->
```

Comment:
I would never use right-handed assignment operators (but might be a personal preference)